smalldata_tools/cube/cube_rank0_utils.py

```diff
@@ -126,9 +126,10 @@ def make_report(anaps, cube_infos, hist_list, filters, varList, exp, run):
                 detname = det['source']
                 if 'thresADU' in det:
                     threshold = det['thresADU']
+                maxHis = getattr(det, 'maxHis', None)
```

Comment:
How is maxHis intended to be used? Will maxHis be a parameter of the det object? 
I have been trying to find a good way to get the pixel histogram meaningul for most cases, with a somewhat underwhelming success...

Reply:
Yeah, I just left that ini because the typically reasonable plotting range did only catch the lower end of the 16 keV gas scattering data. That said, after adding this parameter, I don't think  had yet pushed it all the way through - the idea was that one could put a specific maxHis in letsCube or so to set a range if people know what they want/see the default does not work in this specific case.